s3/replication/replicator/src/s3replicator/app.py

```diff
@@ -24,14 +24,37 @@
 from s3replicationcommon.log import setup_logger
 from s3replicationcommon.jobs import Jobs
 from .replicator_routes import routes
+from .replication_managers import ReplicationManager
+from .replication_managers import ReplicationManagers
 from .session_manager import close_all_sessions
 
 _logger = logging.getLogger('s3replicator')
 
 
+async def on_startup(app):
+    _logger.debug("Starting server...")
+    # Currently only one replication manager is registered.
+    config = app["config"]
+    managers_list = app['replication-managers']
+
+    remote_endpoint = config.get_replication_manager_endpoint()
+    replication_manager = ReplicationManager(remote_endpoint)
+    connected = await replication_manager.subscribe(
+        config.get_replicator_endpoint(), config.max_replications)
+    if connected:
+        _logger.debug("Subscribed successfully for jobs...")
+        managers_list[replication_manager.id] = replication_manager
+    else:
+        await close_all_sessions(app)
+        await replication_manager.close()
+        _logger.debug("Failed to subscribe for jobs...")
```

Comment:
can we make it more verbose? may be 'info'. ???

Reply:
Fixed, changed to error.